client/output.json

```diff
@@ -0,0 +1 @@
"language":"en"},{"path":"tool.Vision","line":321,"file":"/src/game/ui/tools/tools.vue","language":"en"},{"path":"game.ui.tools.tools.change_mode","line":344,"file":"/src/game/ui/tools/tools.vue","language":"en"},{"path":"game.ui.ui.github_msg","line":138,"file":"/src/game/ui/ui.vue","language":"en"},{"path":"game.ui.ui.discord_msg","line":146,"file":"/src/game/ui/ui.vue","language":"en"},{"path":"game.ui.ui.patreon_msg","line":154,"file":"/src/game/ui/ui.vue","language":"en"},{"path":"game.ui.ui.open_loc_menu","line":176,"file":"/src/game/ui/ui.vue","language":"en"},{"path":"open_settings","line":182,"file":"/src/game/ui/ui.vue","language":"en"},{"path":"game.ui.ui.new_ver_msg","line":201,"file":"/src/game/ui/ui.vue","language":"en"},{"path":"game.ui.ui.changelog_RELEASE_LOG","line":203,"file":"/src/game/ui/ui.vue","language":"en"},{"path":"common.version","line":162,"file":"/src/game/ui/ui.vue","language":"en"},{"path":"settings.account.no_pwd_msg","line":43,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.pwd_not_match","line":47,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.server_request_error","line":55,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.common.confirm","line":59,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.change_pwd","line":67,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.remove_account_msg","line":72,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.delete_request_error","line":80,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.no_email_set","line":109,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.general","line":89,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.username","line":91,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.email","line":104,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.danger_zone","line":116,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.new_pwd","line":118,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.repeat_pwd","line":124,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.common.cancel","line":134,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.change_pwd","line":141,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.delete_account","line":147,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.account.delete_confirm_msg","line":150,"file":"/src/settings/account.vue","language":"en"},{"path":"settings.settings.account_settings","line":50,"file":"/src/settings/settings.vue","language":"en"},{"path":"settings.settings.account","line":60,"file":"/src/settings/settings.vue","language":"en"}],"unusedKeys":[]}
```

Comment:
All good, just these keys are missing a translation key in the english locales if I'm not mistaken

Reply:
Deleted, sorry for bringing unnecessary files.

Reply:
But they are missing in en.json right ?

Reply:
Nope, it's just generated by `i18n:report`, and I just resolved all those missing keys.